gcloud-java-datastore/src/main/java/com/google/gcloud/datastore/DatastoreOptions.java

```diff
@@ -94,7 +95,8 @@ Builder normalizeDataset(boolean normalizeDataset) {
 
   private DatastoreOptions(Builder builder) {
     super(DatastoreFactory.class, DatastoreRpcFactory.class, builder);
-    normalizeDataset = builder.normalizeDataset;
+    String systemProjectId = System.getProperty(projectId());
```

Comment:
I think we need to make it more specific.

e.g.
"com.google.gcloud.datastore.DatastoreOptions.skipNormalization.projectId", "true"
and then you can check it with [`Boolean.getBoolean(...)`](https://docs.oracle.com/javase/7/docs/api/java/lang/Boolean.html#getBoolean(java.lang.String)).